app/models/email_template.rb

```diff
@@ -1,7 +1,7 @@
 class EmailTemplate < ApplicationRecord
   before_save :ensure_exactly_one_active_per_kind
 
-  enum kind: { signup: 0, trial: 1, assignment: 2 }
+  enum kind: { signup: 0, trial: 1, assignment: 2, half_year: 3 }
```

Comment:
I am sorry but I am a bit lost, what is the difference between the `assignment:` and the `half year:`?
Maybe 
`enum kind: { signup: 0, trial: 1, assignment: 2, group_assignment: 3 }` or something like that could make more sense, since every half year both assignment and group assignment have feedback and hours.

Reply:
Ah, afair the half year will not be just about assignments. I didn't know what assignment would be, so thats why I added this.

Reply:
The assignment had the same reasoning as the one you have for the half year.